nevergrad/optimization/optimizerlib.py

```diff
@@ -256,6 +257,82 @@ def __init__(
 CMA = ParametrizedCMA().set_name("CMA", register=True)
 DiagonalCMA = ParametrizedCMA(diagonal=True).set_name("DiagonalCMA", register=True)
 
+class _FCMA(base.Optimizer):
+
+    def __init__(
+            self,
+            parametrization: IntOrParameter,
+            budget: Optional[int] = None,
+            num_workers: int = 1,
+            scale: float = 1.0,
```

Comment:
in practice `scale` is used in some experiments but the parametrization is now supposed to hold one as well, so it's mostly a legacy argument, if you opt for a different class, I would advice removing it.

Reply:
Should be consistent with the original CMA. In fact, as long as you don't forward bounds to the algorithm, scale is a very important parameter, should be used in experiments. Same with popsize. Both the original CMA and FCMA support scale and popsize. Should we define both via parametrization?

Reply:
> scale is a very important parameter

Agreed, but it's just a scaling of the space, it's more general than just CMA, hence already handled in the parametrization which normalizes the space. So I think it's redundant, but we can always keep it anyway

> Both the original CMA and FCMA support scale and popsize. Should we define both via parametrization?

Then let's make it possible to define both in the algorithm configuration (aka have them both in the init).

Reply:
There is no default scaling (sigma0) defined for original CMA. If you check

https://github.com/CMA-ES/pycma/blob/master/cma/evolution_strategy.py

`sigma0`
        initial standard deviation.  The problem variables should
        have been scaled, such that a single standard deviation
        on all variables is useful and the optimum is expected to
        lie within about `x0` +- ``3*sigma0``.

you see why: It depends on the scaling of the problem variables. 
FCMA uses default sigma0 = 0.3 which makes sense if bounds are available, since FCMAES scales the variables automatically using the bounds. 
Since Nevergrad doesn't forward bounds to the optimization it actually makes no sense to define a default value for scaling. A single value anyway works only if all variables are scaled equally. If not with CMA you should use
'CMA_stds': 'None  # multipliers for sigma0 in each coordinate
with FCMA it is 'sdevs' which can be a scalar - then it acts like scale/sigma0 - or a vector, then it acts like 'CMA_stds'.

Reply:
Hav to check how parametrization normalizes the space. Could be that 0.3 is a better default than 1.0, see  "is expected to lie within about x0 +- 3*sigma0." here:

https://github.com/CMA-ES/pycma/blob/master/cma/evolution_strategy.py

sigma0
initial standard deviation. The problem variables should
have been scaled, such that a single standard deviation
on all variables is useful and the optimum is expected to
lie within about x0 +- 3*sigma0.

Reply:
One of the very aim of the parametrization is to provide a bridge between the input space of the function (which may be discrete, continuous or a mixture of both, with different scaling etc) and a space which is centered around 0 and has standard deviation 1. So if the problem was correctly parametrized by the user, the solution is indeed supposed to lie in x0 +- 3*sigma0.
Then again, parametrization is harder and more important than most people think, so it's not always true, but the best guess you can make is that the vectors you need to handle in `_internal_ask` and `_internal_tell` are as close as you can know from a centered and reduced variable.

Reply:
Overlooked the importance of the parametrization package. So in transforms.py you define transformations mapping input arguments to a standardized interval. Affine(Transform) rescales (and recenters) an input variable. This means that sigma0=1.0 is the "correct" default value both for the original cma and fcma

Reply:
Most of the transformations do not happen anymore in `transforms.py`, but in `data.py` but yes you got it right ;) and sigma0=1.0 is a "best initial guess" (since I would expect ill-parametrization happens quite often, I'm trying hard to make it simpler and simpler though ;) )